iOS/SonarKit.podspec

```diff
@@ -4,40 +4,37 @@ yogakit_version = '~>1.8'
 
 Pod::Spec.new do |spec|
   spec.name = 'SonarKit'
-  spec.version = '0.0.1'
+  spec.version = '0.0.2'
   spec.license = { :type => 'MIT' }
   spec.homepage = 'https://github.com/facebook/Sonar'
   spec.summary = 'Sonar iOS podspec'
   spec.authors = 'Facebook'
   spec.static_framework = true
   spec.source = { :git => 'https://github.com/facebook/Sonar.git',
-                  :branch=> "master" }
+                  :tag => "SonarKit-v#{spec.version}", }
```

Comment:
The reason for this is because of how Cocoapods retrieves source files when installing a pod. Since a cocoapods repo only hosts the podspec, it grabs the source files by checking out the git repo at a certain commit. For SonarKit `v0.0.1`, it grabs the master branch. The problem with this is that since master is always updating, using SonarKit `v0.0.1` won't be consistent through time, since any time anyone installs the pod, cocoapods will fetch the latest code.

In order to make sure the same source content is always used for a given version, best practice is to specify a git tag to pull from. Tag sources are the default when using `pod lib create`. I used `SonarKit-v0.0.2` instead of `v0.0.2` since there are already tags in this repo for releases of the Sonar desktop app. However, I *do* think I need to specify a `.to_s` for the version here.

More documentation on Cocoapods sources [here](https://guides.cocoapods.org/syntax/podspec.html#source)